integration-tests/resteasy-reactive-kotlin/src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
 quarkus.native.enable-https-url-handler=true
+countries/mp-rest/url=https://restcountries.eu/rest
```

Comment:
I think this should just be an endpoint in the app, not an external service.

Reply:
Okay, I'll change it

Reply:
Fixed